### PR TITLE
Update celery stack to v4

### PR DESCRIPTION
**Pull request by pcreech**
Update celery stack to v4 so we no longer have to maintain code for both
3 and 4

closes #[3407](https://pulp.plan.io/issues/3407)
https://pulp.plan.io/issues/3407

### CODE REVIEW

File: packages/python-amqp/python-amqp.spec
Comment (dralley) [inline packages/python-amqp/python-amqp.spec]: https?  other urls have it

**Review by dralley — APPROVED** [review]
I didn't attempt a really deep review, but just looking over it, everything looks good to me.

**Comment by pcreech** [comment]
Yeah, just waiting on smash results against the new packages to see if any more changes are needed.

File: rel-eng/tito.props
Comment (ekohl) [inline rel-eng/tito.props:7]: This might be me nitpicking, but I'd keep this in a separate commit since it's unrelated to the actual bumping of the celery stack.

**Review by ekohl — COMMENTED** [review]
I see a lot of python3 related specs being removed. Is that intentional?

File: rel-eng/tito.props
Comment (pcreech) [reply rel-eng/tito.props:7]: Might be nitpicky, but definitely fair.  I intended to to such, and it got swept up in the git add . && git commit --amend

Comment (pcreech) [reply rel-eng/tito.props:7]: Updated

File: package_manifest.yaml
Comment (ehelms) [inline package_manifest.yaml]: Nit: You may only want one empty line above this.

**Review by ehelms — APPROVED** [review]
Minor comment. Assuming this all builds as scratches.

**Comment by pcreech** [comment]
Test packages built scratch and release, have not scratch built release packages yet.  Waiting for QE sign-off before doing the latter

[Django](http://koji.katello.org/koji/taskinfo?taskID=86697)
[Vine](http://koji.katello.org/koji/taskinfo?taskID=86450)
[Kombu](http://koji.katello.org/koji/taskinfo?taskID=86449)
[Celery](http://koji.katello.org/koji/taskinfo?taskID=86445)
[Billiard](http://koji.katello.org/koji/taskinfo?taskID=86444)
[AMQP](http://koji.katello.org/koji/taskinfo?taskID=86441)
[Pulp](http://koji.katello.org/koji/taskinfo?taskID=86712)

**Comment by pcreech** [comment]
@ekohl Yes, this is intentional.  These packages are being built for rhel7 only, so I wanted to minimize the surface area in these spec files to just what we support on rhel7

**Comment by pcreech** [comment]
Merging after successful release scratch builds:

[Django](http://koji.katello.org/koji/taskinfo?taskID=86890)
[Vine](http://koji.katello.org/koji/taskinfo?taskID=86889)
[Kombu](http://koji.katello.org/koji/taskinfo?taskID=86888)
[Celery](http://koji.katello.org/koji/taskinfo?taskID=86887)
[Billiard](http://koji.katello.org/koji/taskinfo?taskID=86886)
[AMQP](http://koji.katello.org/koji/taskinfo?taskID=86885)

**Pull request merged**
